### ISSUE TITLE
Person editor - only show unassigned devices in dropdown

### ISSUE THREAD

**Issue by SeanPM5** [home-assistant/home-assistant-polymer#4731]
## The request

In the Person editor (Configuration > Persons), if you edit the person "Steve" and assign `device_tracker.steves_iphone` to him, that device will no longer show up in the "Pick device to track" dropdown next time you edit Steve. A good, helpful feature. So this already works to some extent.

**Problem is, this doesn't apply to other persons**. 

So now when it is Martha's turn to assign devices to her person, she will see ALL device tracker entities, _including those already assigned to Steve_. Home Assistant already knows these devices belong to someone else and should not display them for other persons. 

It's just clutter to scroll through potentially dozens of irrelevant devices when looking for the 3-4 devices you own. And if Martha selected one of Steve's devices by accident it would ruin her presence detection too, which is even more reason to hide these.

Proposal here is that once a device is assigned to a person, it's removed from the "Pick device to track" dropdown for **all persons**. This dropdown should only show currently unassigned devices.

**Comment by iantrich** [comment]
What if you wanted a family "person" that was essentially all devices?

**Comment by SeanPM5** [comment]
That feels like an improper use of Person I think? A person is an individual and it's used for presence detection. If you just toss every device into a fake person it's not really going to provide any valuable information IMO.

If someone wanted a way to see all the devices in their household they could just put them into a group or use the auto-entities card. 

Even though I don't see the use case for having a device assigned to multiple people, it could still be supported, just make it something that has to be entered manually rather than being shown in the dropdown.

**Comment by balloob** [comment]
I agree with Sean. Assigning a device to multiple persons should be considered not supported.
